### PR TITLE
Feature/26 investigate and project structure

### PR DESCRIPTION
**Pull request by alinschuller**
TBD

### CODE REVIEW

**Comment by alinschuller** [comment]
[Investigate and project structure](https://trello.com/c/ZRihLggZ/26-investigate-and-project-structure)

**Comment by alinschuller** [comment]
[Investigate and project structure](https://trello.com/c/ZRihLggZ/26-investigate-and-project-structure)

File: BigriverBookstore-api/BigriverBookstore-api.Server.ServiceModel/Configurations/ApiEntryPointConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+using System;
+using JsonApiFramework.ServiceModel.Configuration;
+
+namespace BigriverBookstore_api.ServiceModel.Configurations
+{
+    public class ApiEntryPointConfiguration : ResourceTypeBuilder<ApiEntryPoint>
```
Comment (alexandru-calinoiu) [inline BigriverBookstore-api/BigriverBookstore-api.Server.ServiceModel/Configurations/ApiEntryPointConfiguration.cs]: You should move this into the web service, your domain models don't actually care that you are going to represent them as json.

**Review by alexandru-calinoiu — COMMENTED** [review]
Look good,

I've put a comment and also figure out why there is no `Id` in the pong resource.

**Pull request merged**